mock-responses/googleai/streaming-failure-prompt-blocked-safety.txt

```diff
@@ -4,5 +4,4 @@ data: {"candidates": [{"content": {"parts": [{"text": "text2"}],"role": "model"}
 
 data: {"candidates": [{"content": {"parts": [{"text": "text3"}],"role": "model"}}],"usageMetadata": {"promptTokenCount": 7,"totalTokenCount": 7,"promptTokensDetails": [{"modality": "TEXT","tokenCount": 7}]},"modelVersion": "gemini-2.0-flash"}

```

Comment:
Just realized... isn't it unexpected for there to be output before the `promptFeedback`?

Reply:
Now that you mention it, absolutely. Checking the vertex version of the file, it does contain a single data chunk with the correct data. https://github.com/FirebaseExtended/vertexai-sdk-test-data/blob/main/mock-responses/vertexai/streaming-failure-prompt-blocked-safety.txt 

Fixed